blendergltf.py

```diff
@@ -990,6 +1012,20 @@ def export_image(image):
 
 
 def export_textures(textures):
+    def check_texture(texture):
+        errors = []
+        if texture.image == None:
+            errors.append('has no image reference')
+        else if texture.image.channels not in [3,4]:
+            errors.append('points to {}-channel image (must be 3 or 4)'.format(texture.image.channels))
+
+        if len(errors) > 0:
```

Comment:
Again, would prefer "if errors" unless there is a good reason to use a length check.